test/testing.cmake

```diff
@@ -38,19 +38,19 @@ mpi_test(file_rw_cube_4 4
   ./file_rw
   ${TEST_DATA_DIR}/cube.msh
   testing_cube_4.ptn
-  full full
+  bfs full
```

Comment:
Do the `full full` tests fail?

Reply:
The full full tests pass but they cause a memory leak because full meshes are not freed even when read from a file.

Reply:
@Angelyr OK.  We should create an issue for this and work on resolving it.